fili-core/src/main/java/com/yahoo/bard/webservice/web/ApiFilter.java

```diff
@@ -30,16 +30,34 @@
      * @param operation  Operation the filter operates with
      * @param values  The values the filter uses when operating
      */
+    @Deprecated
```

Comment:
Should include some JavaDoc briefly explaining why this was deprecated, and 
what alternative people should use.